bears/r/FormatRBear.py

```diff
@@ -15,8 +16,38 @@ class FormatRBear:
     LANGUAGES = "R"
 
     @staticmethod
-    def create_arguments(filename, file, config_file):
-
-        rcode = ("formatR::tidy_source(source =\""+filename+"\"")
+    def create_arguments(filename, file, config_file,
+                         r_comment: bool = True,
+                         r_blank: bool = True,
+                         r_brace: bool = False,
+                         r_arrow: bool = False,
```

Comment:
I think these could be more descriptive, like `r_keep_blank_lines` or `r_keep_comments`